remix-analyzer/src/solidity-analyzer/modules/functionCallGraph.js

```diff
@@ -80,20 +80,20 @@ function resolveCallGraphSymbol (callGraph, funcName) {
 }
 
 function resolveCallGraphSymbolInternal (callGraph, funcName, silent) {
-  var current
+  let current
   if (funcName.includes('.')) {
-    var parts = funcName.split('.')
-    var contractPart = parts[0]
-    var functionPart = parts[1]
-    var currentContract = callGraph[contractPart]
+    const parts = funcName.split('.')
+    const contractPart = parts[0]
+    const functionPart = parts[1]
```

Comment:
I don't know how much you want to reformat: 
`const [ contractPart, functionPart ] = funcName.split('.')`